setup.py

```diff
@@ -2,13 +2,13 @@
 
 setup(
     name="m2cgen",
-    version="0.1.2",
+    version="0.1.3",
```

Comment:
The `0.1.2` hasn't been released yet, so it doesn't make sense to bump the minor version. However, I think this update deserves to be `0.2.0` so why don't we make it so? :)